src/plugins/Transloadit/index.js

```diff
@@ -87,11 +87,25 @@ module.exports = class Transloadit extends Plugin {
 
   getAssemblyOptions (fileIDs) {
     const options = this.opts
+
+    const normalizeAssemblyOptions = (assemblyOptions) => {
+      const globalMeta = this.uppy.getState().meta
+      // Include global metadata as fields. Works great together with the Form plugin :)
+      if (assemblyOptions.fields === true) {
+        assemblyOptions.fields = Object.assign({}, globalMeta)
+      }
+      if (!assemblyOptions.fields) {
+        assemblyOptions.fields = {}
+      }
+      return assemblyOptions
+    }
+
     return Promise.all(
       fileIDs.map((fileID) => {
         const file = this.uppy.getFile(fileID)
         const promise = Promise.resolve()
           .then(() => options.getAssemblyOptions(file, options))
+          .then(normalizeAssemblyOptions)
```

Comment:
Nitpicking here: is `normalizeAssemblyOptions` a good name when it mostly adds `fields` to an assembly?

Reply:
My thinking was that the fact that it mostly does fields is an implementation detail: this function takes user provided options that may use shorthands etc. and returns options in a normalized single format for the plugin.

Reply:
Yeah, ok, makes sense.